File: 2413-smallest-even-multiple/2413-smallest-even-multiple.cpp
class Solution {
public:
    int smallestEvenMultiple(int n) {
        int temp;
        for(int i = 1; i<=n; i++){
            if((n*i)%2==0){
              temp = n*i;
               break;  
            }
        }   return temp;
    }
};

File: 2413-smallest-even-multiple/2413-smallest-even-multiple.java
class Solution {
    public int smallestEvenMultiple(int n) {
        int temp = 0;
        for(int i = 1; i <= n*2; i++) {
            if(i % 2 == 0 && i % n == 0) {
                temp = i;
                break;
            }
        }
        return temp;
    }
}